packages/codefixes/test/__snapshots__/supressed-errors.test.ts.snap

```diff
@@ -0,0 +1,48 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Supressed Errors > TS2339: False expression: Token end is child end, supressed-errors/missingMethod.ts 1`] = `
+"// @ts-expect-error @rehearsal TODO TS2307: Cannot find module 'some-package' or its corresponding type declarations.
+import { Base } from \\"some-package\\";
```

Comment:
wondering in the instance when ts *can* find the module `some-package` are we still seeing this error?

Reply:
In learning-web, I updated a tsconfig to resolve imports to that package, and it still failed. Despite being able to click through the package name to get to the implementation.

This problem is less about the actual class and something breaking given the method not found on the class, and the large size of the pojo.